staff/gerard-celma/e-commerce-protos/scripts/cart.test.js

```diff
@@ -0,0 +1,116 @@
+var cart = new Cart;
+
+var socks = new Socks('Calvin Klein', 42, 'black', 9.99);
+var tanga = new Tanga('Wicked Weasel', 32, 'red', 29.95);
+var bra = new Bra('Calvin Klein', 'push-up', 80, 'cream', 39); // model
+var slips = new Slips('Abanderado', 42, 'purple', 13.99);
+
+cart.add(socks);
+cart.add(tanga);
+cart.add(bra);
+cart.add(slips);
+
+
+suite('cart', function() {
+
+    describe('Testing cart.add()', function() {
```

Comment:
Maybe I would say testing add method, cause is the suit, you've already said that you are testing the Cart. Anyway, that's a matter of taste 😄